### ISSUE TITLE
Is it possible to check status of cron it has executed the process

### ISSUE THREAD

**Issue by parthpkp97** [node-cron/node-cron#199]
I have a cronjob that i want to run every 30 minutes and i need to check status if that cron job has completed its task.

Right now there i a method getStatus but it only gives two types of status scheduled and running I want to get status if it has completed it's task.

Actually i want to stop cron job if it is taking more than 10 minutes time to execute that job.

**Comment by alessandrorubio** [comment]
Yes, it would be useful to know get some custom output when the task is ended.

**Comment by jon-malave** [comment]
@parthpkp97 Yes there is an undocumented method for this. Just use `.getStatus()`

**Comment by mithun1999** [comment]
@jon-malave The method is not working. It's returning error.

```
job.getStatus is not a function
```

**Comment by isbabaev** [comment]
@mithun1999 I have the same problem

**Comment by ngloom** [comment]
@jon-malave  just got the same error as @mithun1999  with node-cron 3.0.0
